src/Disciplina/Messages.hs

```diff
@@ -0,0 +1,54 @@
+
+-- | Starting point for running a Witness node
+
+module Disciplina.Messages where
+
+import Universum
+
+import           Data.Binary (Binary)
```

Comment:
Broken indent. Are you sure that you use latest Serokell `stylish-haskell.yaml`? Here it is: https://github.com/serokell/serokell-util/blob/master/.stylish-haskell.yaml